mixins/form.jsx

```diff
@@ -150,10 +150,20 @@ module.exports = {
       },
       body: JSON.stringify(props)
     }).then(function(response) {
+      if (!response.ok) {
+        throw new Error(response.statusText);
+      }
+      if (!response.headers.get("content-type")) {
```

Comment:
The above fetch is shared with signup, paypal, and stripe.

Paypal and stripe return json, and signup returns nothing.

This is why I'm doing this check.